src/index.ts

```diff
@@ -23,6 +25,8 @@ export { Element, ElementEvents, Link, LinkEvents } from './ontodia/diagram/elem
 export { LayoutData, LayoutCell, LayoutElement, LayoutLink } from './ontodia/diagram/layoutData';
 export * from './ontodia/diagram/model';
 export * from './ontodia/diagram/view';
+export { Cell } from './ontodia/diagram/paper';
+export { PaperArea, PointerEvent, PointerUpEvent } from './ontodia/diagram/paperArea';
```

Comment:
I think we should only export `PointerEvent` and `PointerUpEvent` here because `PaperArea` already being exported in the `InternalApi` namespace.